dts/arm/st/h7/stm32h723.dtsi

```diff
@@ -91,6 +91,15 @@
 			status = "disabled";
 		};
 
+		dcmi: dcmi@48020000 {
```

Comment:
Sorry for the late catch, but I've just had a check and dcmi is actually available on the whole h7 series, so you can directly put the node in the common file of the series stm32h7.dtsi and remove other references

Reply:
No worries! Updated.
Thank you for your valuable review! :)